src/components/PopupWithForm.js

```diff
@@ -5,7 +5,17 @@ export default class PopupWithForm extends Popup {
     this._popupForm = this._popupElement.querySelector(".modal__form");
     this._handleFormSubmit = handleFormSubmit;
     this._modalInputs = this._popupForm.querySelectorAll(".modal__input");
+    this._submitButton = this._popupForm.querySelector(".modal__button");
+    this._submitButtonText = this._submitButton.textContent;
   }
+renderLoading(isLoading, loading = 'Saving...') {
```

Comment:
Needs correction: It's important to format code correctly, use the indentation hierarchy, and split your code. Please check these sources to study the standard: [https://code.s3.yandex.net/web-developer/landings/design-rules-en/index.html](https://code.s3.yandex.net/web-developer/landings/design-rules-en/index.html), [https://javascript.info/coding-style](https://javascript.info/coding-style).

To make sure your code is formatted correctly, we suggest using Prettier (https://prettier.io/). It can be integrated with any code editor or IDE.

You should fix the formatting in all the files